CONTRIBUTING.md

```diff
@@ -1,14 +1,110 @@
-## Contributing to expressjs.com
+# Contributing to Expressjs.com
 
-This repository is only for issues related to the website [http://expressjs.com](http://expressjs.com). For issues related to Express, the framework, go to [https://github.com/expressjs/express](https://github.com/expressjs/express).
+### The Official Documentation of the Express Framework
 
-Feel free to make changes to the template files or the document files. The supporting docs are located in their respective directories, and the API docs are located under the `_includes` directory.
+>[!NOTE]
```

Comment:
Okay I made this into a GH tag and it gets filtered out for the site. 
I won't make any more changes other than from reviewers

Reply:
sure